src/goReferences.ts

```diff
@@ -40,7 +40,7 @@ export class GoReferenceProvider implements vscode.ReferenceProvider {
 				try {
 					if (err && (<any>err).code === 'ENOENT') {
 						vscode.window.showInformationMessage('The "guru" command is not available.  Use "go get -v golang.org/x/tools/cmd/guru" to install.', 'Install').then(selected => {
-							installTool('guru');
+							installTools(['guru']);
```

Comment:
FWIW -There's another open issue tracking adding this logic to all of the places where we put up an error about a tool not being available.  Would be great to address this as well either as part of this change or as a follow-on change.